TESTS/mbed_drivers/timeout/main.cpp

```diff
@@ -68,7 +68,8 @@ Case cases[] = {
     Case("1 s delay during sleep (attach_us)", test_sleep<AttachUSTester<Timeout>, 1000000, LONG_DELTA_US>,
          greentea_failure_handler),
 #endif
-#if !defined(__ARM_FM)  //FastModels not support time drifting test
+
+#if !defined(__ARM_FM) && !defined(SKIP_TIME_DRIFT_TESTS) //FastModels not support time drifting test
```

Comment:
the macro `__ARM_FM` was only used for FastModels skip the time drifting tests. you can completely replace it by `SKIP_TIME_DRIFT_TESTS`. they are in several files, and https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L8033

Reply:
To be sure that I correctly understood. Should I completely remove  `__ARM_FM` macro?

Reply:
yes please, you can do a search and replace all the `__ARM_FM` with `SKIP_TIME_DRIFT_TESTS`. basically they are in every file you created the macro